releasenotes/notes/fix-qi-transpiled-8df449529bf6d9a2.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fix :meth:`~qiskit.utils.QuantumInstance.execute` method in the 
+    :class:`~qiskit.utils.QuantumInstance` to not modify the transpiled input circuit.
+    Refer to the discussion in `#7449 <https://github.com/Qiskit/qiskit-terra/issues/7449>`__.
```

Comment:
I think it never modified the circuits, right, only the list of circuits passed in?  The release note sounds like it was changing the circuits themselves.

Reply:
Yes, only the array, not the circuits